server.ts

```diff
@@ -0,0 +1,35 @@
+///<reference path='.\Declarations\node\node.d.ts' />
```

Comment:
And a general comment, attaching to `server.ts` but applicable to any: might be worth placing the MIT license header on each file.